clustertool/embed/generic/root/DOTREPLACEdevcontainer/postCreateCommand.sh

```diff
@@ -12,6 +12,22 @@ fisher install nickeb96/puffer-fish
 fisher install PatrickF1/fzf.fish
 "
 
+sudo apk add sshfs
+
+(
+  set -x; cd "$(mktemp -d)" &&
+  OS="$(uname | tr '[:upper:]' '[:lower:]')" &&
+  ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/')" &&
+  KREW="krew-${OS}_${ARCH}" &&
+  curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/latest/download/${KREW}.tar.gz" &&
+  tar zxvf "${KREW}.tar.gz" &&
+  ./"${KREW}" install krew
+)
+
+export PATH="${KREW_ROOT:-$HOME/.krew}/bin:$PATH"
+
+kubectl krew install pv-mounter
```

Comment:
Cannot this be done on container build time?
I would prefer that.

Reply:
And as it is a temporary solution until we make our own container. I think its fine

Reply:
> And as it is a temporary solution until we make our own container. I think its fine

We already make our own container.

Reply:
Just because we use an upstream, doesn't mean we dont build our own container.
Just like we use an alpine upstream for example.

Reply:
working on it